sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/fabiomestre/unified-runtime.git")
```

Comment:
this doesn't seem right

Reply:
It is pointing to the fork because the UR PR is not merged yet. Once that PR is merged, this code will be updated to point to it.

However, I think that it doesn't make sense to merge the PR on the UR side before the changes here have been reviewed.